test/typescript/replaceReducer.ts

```diff
@@ -1,26 +0,0 @@
-import { combineReducers, createStore } from '../..'
-
-/**
- * verify that replaceReducer maintains strict typing if the new types change
- */
-const bar = (state = { value: 'bar' }) => state
-const baz = (state = { value: 'baz' }) => state
-const ACTION = {
-  type: 'action'
-}
-
-const originalCompositeReducer = combineReducers({ bar })
-const store = createStore(originalCompositeReducer)
-store.dispatch(ACTION)
-
-const firstState = store.getState()
-firstState.bar.value
-// @ts-expect-error
-firstState.baz.value
-
-const nextStore = store.replaceReducer(combineReducers({ baz })) // returns ->  { baz: { value: 'baz' }}
-
-const nextState = nextStore.getState()
-// @ts-expect-error
-nextState.bar.value
-nextState.baz.value
```

Comment:
Removed this test as well since it was written to test the idea that `replaceReducer` would return a store with new types, which is no longer the case.